chain/bitcoind_conn.go

```diff
@@ -147,9 +147,29 @@ func NewBitcoindConn(cfg *BitcoindConfig) (*BitcoindConn, error) {
 	}
 
 	// Verify that the node is running on the expected network.
-	net, err := getCurrentNet(client)
-	if err != nil {
-		return nil, err
+	var (
+		net wire.BitcoinNet
+	)
```

Comment:
Nit: this can be collapsed into a single line `var net wire.BitcoinNet`.

Reply:
Thanks, a leftover from when this also declared `err` which didn't exactly work any more since it's being used above.